python/adjoint/objective.py

```diff
@@ -284,11 +293,30 @@ def __call__(self):
 
 
 class FourierFields(ObjectiveQuantity):
+    """A differentiable frequency-dependent Fourier Fields (dft_fields)
+    Attributes:
+        volume: the volume over which the Fourier Fields are calculated. Due to an unsolved bug,
+          the size must not be zero in at least one direction.
+        component: field component (e.g. mp.Ex, mp.Hz, etc.)
+        yee_grid: if True, the Fourier fields evaluated at corresponding Yee grid point
+          are returned; otherwise, interpolated fields at the center of each voxel
+          are returned
+        decimation_factor: an integer so that the DFT fields are updated at every
```

Comment:
An integer used to specify the number of timesteps between updates of the DFT fields.